appveyor.yml

```diff
@@ -20,16 +20,45 @@ install:
 
     choco install Firefox | out-null
 
-- cmd: >-
-    node --version
-
     npm --version
 
-    npm install
-
+    npm i -s
```

Comment:
Why using `-s/--silent` here? It hides NPM error messages, that makes debugging harder.

Reply:
Agree. It should be `npm i --loglevel error`.